docs/configuration-reference/components/contour.md

```diff
@@ -48,6 +70,8 @@ Example:
 |------------------|---------------------------------------------------------------------------------------------|:-------:|:--------:|
 | `service_monitor`| Create ServiceMonitor for Prometheus to scrape Contour and Envoy metrics.                   | false   | false    |
 | `ingress_hosts`  | [ExternalDNS component](external-dns.md) creates DNS entries from the values provided.      | ""      | false    |
+| `node_affinity`  | Node Affinity for deploying the operator pod and envoy daemonset.                           | -       | false    |
```

Comment:
Could you please fix the capitalization (throughout the PR) according to our style guide?

Reply:
changed.